src/scheduler/data_types.h

```diff
@@ -470,6 +472,8 @@ struct job_info
 
 	struct attrl *attr_updates;	/* used to federate all attr updates to server*/
 	float formula_value;		/* evaluated job sort formula value */
+	nspec **resreleased;		/* list of resources released by the job on each node */
```

Comment:
Okay, if that is the case, the external design needs updating, because it says: "It stores a string that depicts the amount of resources that are released on each chunk that the job was running on"